js/src/Ice/EndpointFactoryManager.js

```diff
@@ -69,7 +69,7 @@ export class EndpointFactoryManager {
                     // and ask the factory to read the endpoint data from that stream to create
                     // the actual endpoint.
                     //
-                    const os = new OutputStream(this._instance, Protocol.currentProtocolEncoding);
+                    const os = new OutputStream(Protocol.currentProtocolEncoding);
```

Comment:
I think currentProtocolEncoding is incorrect here (is/was same in other mappings). It should be Ice.currentEncoding, i.e. 1.1. Or nothing since currentEncoding is the default encoding.

We use currentProtocolEncoding when creating a protocol message. But that's not what we're doing here.